threads-api/src/threads-api.ts

```diff
@@ -286,4 +296,101 @@ export class ThreadsAPI {
     const likers = res.data.data.likers;
     return likers;
   };
+
+  getToken = async (): Promise<string | undefined> => {
+    if (!this.username || !this.password) {
+      return;
+    }
+
+    const base = 'https://i.instagram.com/api/v1/';
+    const url = `${base}bloks/apps/com.bloks.www.bloks.caa.login.async.send_login_request/`;
+    const blockVersion = '5f56efad68e1edec7801f630b5c122704ec5378adbee6609a448f105f34a9c73';
+
+    const params = encodeURIComponent(
+      JSON.stringify({
+        client_input_params: {
+          password: this.password,
+          contact_point: this.username,
+          device_id: `android-${this.androidId}`,
```

Comment:
Wanna add the `device-` prefix to `this.androidId` itself?
ref: https://github.com/m1guelpf/threads-re#authentication

Reply:
Yeah, much better now! I renamed `androidId` to `deviceId` too

Reply:
👍👍👍 Looks great!